packages/components/app/styles/components/application-state.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APPLICATION-STATE COMPONENT
+//
+$hds-application-state-padding: 12px 0;
+
+.hds-application-state {
+  align-self: center;
+  width: 20rem;
+  max-width: 100%;
+  margin: 0 auto;
+}
+
+.hds-application-state__header,
```

Comment:
I would delete ".hds-application-state__header," here since you override the display below.